### PR TITLE
feat(BrandPositioning): Add settings

### PR DESCRIPTION
**Pull request by leko91**

### CODE REVIEW

File: packages/brand-positioning-block/src/settings.ts [hunk 1]
```diff
@@ -1,18 +1,438 @@
...
+                    label: 'Axis labels',
+                    defaultValue: BOARD_AXIS_LABELS_COLOR_DEFAULT_VALUE,
+                },
+                {
+                    id: 'boardBackgroundColor',
+                    type: 'colorInput',
+                    label: 'Background',
+                    defaultValue: BOARD_BACKGROUND_COLOR_DEFAULT_VALUE,
+                },
+                {
```
Comment (fulopdaniel) [inline packages/brand-positioning-block/src/settings.ts:226]: For borders and border radiuses you can reuse `getBorderRadiusSettings, getBorderSettings` from the `@frontify/guideline-blocks-shared` package

File: packages/brand-positioning-block/src/settings.ts [hunk 2]
```diff
@@ -1,18 +1,438 @@
...
+                                    type: 'colorInput',
+                                    defaultValue: BRAND_ITEM_BORDER_COLOR_DEFAULT_VALUE,
+                                },
+                            ],
+                            layout: MultiInputLayout.Columns,
+                        },
+                    ],
+                },
+                {
+                    id: 'isBrandIteCornerRadiusCustom',
```
Comment (fulopdaniel) [inline packages/brand-positioning-block/src/settings.ts:389]: Typo, but also if you reuse the helper functions it will resolve

**Review by fulopdaniel — APPROVED** [review]

File: packages/brand-positioning-block/src/settings.ts [hunk 3]
```diff
@@ -1,18 +1,438 @@
...
+    MultiInputLayout,
+    appendUnit,
+    defineSettings,
+    numericalOrPixelRule,
+    presetCustomValue,
+} from '@frontify/guideline-blocks-settings';
+
+import { brandItemBrandItemSizeMap, cornerRadiusMap } from './types';
+
+const WHITE_COLOR_VALUE: Color = {
```
Comment (Kenny806) [inline packages/brand-positioning-block/src/settings.ts:17]: Can't we use fondue tokens for the colors?

Comment (leko91) [reply packages/brand-positioning-block/src/settings.ts:17]: Not that I'm aware of. Not sure even if these colors exist there 😄

**Review by Kenny806 — APPROVED** [review]

**Pull request merged**